bql/planner/planner_test.go

```diff
@@ -983,6 +983,130 @@ func TestPlannerQuery(t *testing.T) {
 			nBindings: 2,
 			nRows:     5,
 		},
+		{
```

Comment:
Yes, the test you discussed in your previous comment comes exactly for that. The query there in `planner_test.go` is:

```
SELECT ?s, ?p, ?o
FROM ?test
WHERE {
  ?s ?p ?o .
  FILTER latest(?p)
};
```

Since in our `?test` graph we have the temporal triples below (the immutable ones will all be skipped by the `latest` `FILTER`):

```
/u<peter> "bought"@[2016-01-01T00:00:00-08:00] /c<mini>
/u<peter> "bought"@[2016-02-01T00:00:00-08:00] /c<model s>
/u<peter> "bought"@[2016-03-01T00:00:00-08:00] /c<model x>
/u<peter> "bought"@[2016-04-01T00:00:00-08:00] /c<model y>

/u<paul> "bought"@[2016-01-01T00:00:00-08:00] /c<model n>
/u<paul> "bought"@[2016-04-01T00:00:00-08:00] /c<model r>

/item/book<000> "in"@[2016-04-10T4:21:00.000000000Z] /room<Hallway>
/item/book<000> "in"@[2016-04-10T4:23:00.000000000Z] /room<Kitchen>
/item/book<000> "in"@[2016-04-10T4:25:00.000000000Z] /room<Bedroom>
```

We have `/u<peter> "bought"@[2016-04-01T00:00:00-08:00] /c<model y>` and `/u<paul> "bought"@[2016-04-01T00:00:00-08:00] /c<model r>` with the same predicate, and both have the same latest anchor (`2016-04-01T00:00:00-08:00`) for the predicate ID `"bought"`. This way, we expect both to to appear in the table result. I added the later triple with `/u<paul>` to precisely test that, as commented in that commit description.

Then, the final result table expected for the query above is:

```
?s		 ?p					?o
/u<peter>	 "bought"@[2016-04-01T00:00:00-08:00]	/c<model y>
/u<paul>	 "bought"@[2016-04-01T00:00:00-08:00]	/c<model r>
/item/book<000>	 "in"@[2016-04-10T04:25:00Z]		/room<Bedroom>
```

Testing if `FILTER latest` is indeed returning the two temporal triples with the same latest timestamp `2016-04-01T00:00:00-08:00`.